### PR TITLE
Use localize in the Supervisor panel

### PR DESCRIPTION
**Pull request by ludeeus**
## Breaking change

Converted most of the Supervisor panel to use localize for text, some dialogs are not converted, there are plans to work on them first.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/addon-view/config/hassio-addon-audio.ts [hunk 1]
```diff
@@ -48,12 +51,16 @@ class HassioAddonAudio extends LitElement {
 
   protected render(): TemplateResult {
     return html`
-      <ha-card header="Audio">
+      <ha-card
+        header="${this.supervisor.localize("addon.configuration.audio.header")}"
```
Comment (bramkragten) [inline hassio/src/addon-view/config/hassio-addon-audio.ts:55]:
```suggestion
        .header="${this.supervisor.localize("addon.configuration.audio.header")}"
```

File: hassio/src/addon-view/config/hassio-addon-audio.ts [hunk 2]
```diff
@@ -48,12 +51,16 @@ class HassioAddonAudio extends LitElement {
...
+      >
         <div class="card-content">
           ${this._error ? html` <div class="errors">${this._error}</div> ` : ""}
 
           <paper-dropdown-menu
-            label="Input"
+            label="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/config/hassio-addon-audio.ts:61]:
```suggestion
            .label=${this.supervisor.localize(
```

File: hassio/src/addon-view/config/hassio-addon-audio.ts [hunk 3]
```diff
@@ -64,15 +71,17 @@ class HassioAddonAudio extends LitElement {
...
+                  <paper-item device=${item.device || ""}>
+                    ${item.name}
+                  </paper-item>
                 `;
               })}
             </paper-listbox>
           </paper-dropdown-menu>
           <paper-dropdown-menu
-            label="Output"
+            label="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/config/hassio-addon-audio.ts:82]:
```suggestion
            .label=${this.supervisor.localize(
```

File: hassio/src/addon-view/config/hassio-addon-config.ts [hunk 4]
```diff
@@ -127,12 +138,20 @@ class HassioAddonConfig extends LitElement {
...
+                    "addon.configuration.options.invalid_yaml"
+                  )}
+                </div>
+              `}
         </div>
         ${hasHiddenOptions
           ? html`<ha-formfield
               class="show-additional"
-              label="Show unused optional configuration options"
+              label="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/config/hassio-addon-config.ts:152]:
```suggestion
              .label=${this.supervisor.localize(
```

File: hassio/src/addon-view/config/hassio-addon-network.ts [hunk 5]
```diff
@@ -55,16 +58,30 @@ class HassioAddonNetwork extends LitElement {
     }
 
     return html`
-      <ha-card header="Network">
+      <ha-card
+        header="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/config/hassio-addon-network.ts:62]:
```suggestion
        .header=${this.supervisor.localize(
```

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 6]
```diff
@@ -176,14 +119,28 @@ class HassioAddonInfo extends LitElement {
     return html`
       ${this.addon.update_available
         ? html`
-            <ha-card header="Update available! 🎉">
+            <ha-card
+              header="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/info/hassio-addon-info.ts:123]:
```suggestion
              .header=${this.supervisor.localize(
```

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 7]
```diff
@@ -305,7 +284,9 @@ class HassioAddonInfo extends LitElement {
                       })}
                       @click=${this._showMoreInfo}
                       id="stage"
-                      label="stage"
+                      label="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/info/hassio-addon-info.ts:287]:
```suggestion
                      .label=${this.supervisor.localize(
```

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 8]
```diff
@@ -331,7 +312,9 @@ class HassioAddonInfo extends LitElement {
                       <ha-label-badge
                         @click=${this._showMoreInfo}
                         id="host_network"
-                        label="host"
+                        label="${this.supervisor.localize(
```
Comment (bramkragten) [inline hassio/src/addon-view/info/hassio-addon-info.ts:315]:
```suggestion
                        .label=${this.supervisor.localize(
```

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 9]
```diff
@@ -712,8 +736,21 @@ class HassioAddonInfo extends LitElement {
...
     showHassioMarkdownDialog(this, {
-      title: PERMIS_DESC[id].title,
-      content: PERMIS_DESC[id].description,
+      title: this.supervisor.localize(`addon.dashboard.capability.${id}.title`),
+      content:
+        id === "stage"
+          ? this.supervisor.localize(
+              `addon.dashboard.capability.${id}.description`,
+              "icon_stable",
+              `<ha-svg-icon path="${STAGE_ICON.stable}"></ha-svg-icon>`,
```
Comment (bramkragten) [inline hassio/src/addon-view/info/hassio-addon-info.ts:745]:
```suggestion
              html`<ha-svg-icon .path=${STAGE_ICON.stable}></ha-svg-icon>`,
```

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 10]
```diff
@@ -859,12 +906,14 @@ class HassioAddonInfo extends LitElement {
...
       showHassioMarkdownDialog(this, {
-        title: "Changelog",
+        title: this.supervisor.localize("addon.dashboard.changelog"),
         content,
       });
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to get addon changelog",
+        title: this.supervisor.localize(
+          "addon.dashboard.action_error.get_chnagelog"
```
Comment (bramkragten) [inline hassio/src/addon-view/info/hassio-addon-info.ts:915]:
```suggestion
          "addon.dashboard.action_error.get_changelog"
```

File: hassio/src/addon-view/info/hassio-addon-info.ts (hunk 9 above)
Comment (ludeeus) [reply hassio/src/addon-view/info/hassio-addon-info.ts:745]: That will not work.

**Review by bramkragten — APPROVED** [review]

File: hassio/src/addon-view/info/hassio-addon-info.ts (hunk 9 above)
Comment (ludeeus) [reply hassio/src/addon-view/info/hassio-addon-info.ts:745]: showHassioMarkdownDialog does not support it, will handle that later

**Pull request merged**